### DISCUSSION TITLE
Calculating AES throughput

### DISCUSSION THREAD

**Discussion by Yzgll** [sarchlab/mgpusim#57] [Q&A]
Hello author, after solving the AES length issue last time, I encountered new difficulties again. After reviewing the performance metrics on Daisen, I found myself at a loss when calculating the throughput of AES. I didn't know how to utilize Daisen's parameters. I have considered dividing the length by the total time or kernel time to obtain throughput, but I think this is more biased towards algorithm level throughput rather than hardware level throughput. I don't know what to do. Can you give me some advice?

**Comment by syifan** [comment]
How do you define hardware-level throughput?

**Reply by Yzgll** [reply]
I think it's the amount of work done per unit of time. That's why I was thinking about the total data which is the length divided by the total time.

**Reply by syifan** [reply]
OK. I still think you do not really have to use Daisen to get the information. You can use the data in the `metrics.csv` file. I suggest you to calculate with `Length/KernelTime/NumCU`. If you have not changed the configuration, the default MGPUSim configuration has 64 CUs per GPU.

**Reply by Yzgll** [reply]
Thank you！
